lib/fakeout.rb

```diff
@@ -15,21 +14,22 @@ def initialize
     # create products (can be free)
     def events(count = 1, options = {})
       1.upto(count) do
-        attributes   = { :title        => Faker::Commerce.product_name,
-                         :description => Faker::Lorem.paragraph(2),
-                         :location => Faker::Address.city,
-                         :start_date => Faker::Time.between(DateTime.now - 5, DateTime.now),
-                         :end_date => Faker::Time.between(DateTime.now, DateTime.now + 5),
-                         :image => Faker::Avatar.image,
-                         :venue => Faker::Address.street_address,
-                         :category => pick_random(Category),
-                         :event_manager => pick_random(User),
-                         :event_template => pick_random(EventTemplate)
-                          }.merge(options)
-        event      = Event.new(attributes)
+        attributes = {
+          title: Faker::Commerce.product_name,
+          description: Faker::Lorem.paragraph(2),
+          location: Faker::Address.city,
+          start_date: Faker::Time.between(DateTime.now - 5, DateTime.now),
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.